openwisp_users/admin.py

```diff
@@ -204,7 +184,7 @@ def get_inline_instances(self, request, obj=None):
         if request:
             if hasattr(inline, '_has_add_permission'):
                 has_add_perm = inline._has_add_permission(request, obj)
-            else:
+            if hasattr(inline, 'has_add_permission'):
```

Comment:
Is this possible?
```
if hasattr(inline, '_has_add_permission') or hasattr(inline, 'has_add_permission'):
```

Reply:
I tried doing that but i noticed inline can have `_has_add_permission` and/or `has_add_permission` in some situations. thus two conditions will be needed to make `inline.has_add_permission` and `inline._has_add_permission` to dynamically take care of the situations.
I Equally tried doing this;
```
if request:
    has_add_perm=inline.has_add_permission(request)
    if hasattr(inline, '_has_add_permission'):
        has_add_perm = inline._has_add_permission(request, obj)
```
which will override the value of `has_add_perm` if `_has_add_permission` exist since i noticed priority was given to the `_has_add_permission` attribute.
But that didn't increase the coverage.

Reply:
I do not understand the nature of the change on this line, we should check the django documentation or source to understand if we can do this, @NoumbissiValere is there a chance you did find out?

As far as I know, the old code was working fine and I would not change it unless it really makes sense to do it.
Having stable and predictable code has the priority over test coverage.